### ISSUE TITLE
bad translateTime/localTime output

### ISSUE THREAD

**Issue by victor0801x** [pinojs/pino-pretty#7]
1. I clone this repo and run its tests are correct, but use pino-pretty as global cli will get this weird output.
```bash
$ echo '{"level":30,"time":1522431328992,"msg":"hello world","pid":42,"hostname":"foo","v":1}' |pino-pretty --colorize --translateTime
[2018-35-30 17:03:28.ththth UTC] INFO (42 on foo): hello world

$ echo '{"level":30,"time":1522431328992,"msg":"hello world","pid":42,"hostname":"foo","v":1}' |pino-pretty --colorize --translateTime --localTime
[2018-35-31 01:03:28.ststst CST] INFO (42 on foo): hello world

$ echo '{"level":30,"time":1522431328992,"msg":"hello world","pid":42,"hostname":"foo","v":1}' |pino-pretty --colorize --translateTime --localTime --dateFormat 'yyyy-mm-dd HH:MM:ss.l o'
[2018-03-31 01:35:28.992 +0800] INFO (42 on foo): hello world
```
![2018-04-07 14-37-20](https://user-images.githubusercontent.com/4534038/38452277-77768838-3a73-11e8-9d1e-12f8b2a16c18.png)

2. (not a bug) if we pass `dateFormat` or `localTime` args, shoud we implicit enable `translateTime` ?

**Comment by mcollina** [comment]
I'm not sure what you are proposing. Can you make an example before/after your proposal?

**Comment by victor0801x** [comment]
@mcollina first question is I got weird output when using pino-pretty as global cli.
second, we can just pass `dateFormat` or `localTime` that will translatetime too.
```bash
echo '{"level":30,"time":1522431328992,"msg":"hello world","pid":42,"hostname":"foo","v":1}' | pino-pretty

// current
--localTime // [1522431328992] INFO (42 on foo): hello world
--dateFormat '..' // [1522431328992] INFO (42 on foo): hello world
-t --localTime // [2018-03-31 01:35:28.992 +0800] INFO (42 on foo): hello world
-t --dateFormat '..' // [2018-03-31 01:35:28] INFO (42 on foo): hello world

// proposing, don't need to pass -t/--translateTime when 
--localTime // [2018-03-31 01:35:28.992 +0800] INFO (42 on foo): hello world
--dateFormat '..' // [2018-03-31 01:35:28] INFO (42 on foo): hello world

**Comment by jsumners** [comment]
Thank you for catching the error. I have resolved it in v1.0.1. I also went ahead and made `dateFormat` imply `translateTime` if a user value has been specified for `dateFormat` (must be different from the default value).
